alfresco-authentication/backend/gradle.properties

```diff
@@ -0,0 +1,8 @@
+kotlinVersion=1.7.20
+springVersion=2.7.6
+springDependencyManagementVersion=1.0.11.RELEASE
+valtimoVersion=10.7.0.RELEASE
```

Comment:
It looks like the frontend part of this plugin uses a different version: `10.6.0`

Reply:
updated the frontend to 10.7.0